connectors/tests/test_byoei.py

```diff
@@ -543,6 +570,20 @@ async def setup_fetcher(basic_rule_engine, existing_docs, queue, sync_rules_enab
             deleted(0),
             total_downloads(0),
         ),
+        (
+            # content_extraction_enabled is false,
+            # indexing should still work, but nothing should be downloaded
+            [],
+            [(DOC_ONE, None)],
```

Comment:
I think here you should pass the function for download so that you had a setup where there is a download available, but it's turned off.

Reply:
Yup, that was the one, thanks!